quick_start/index.rst

```diff
@@ -1,12 +1,17 @@
 .. _quick_start_index:
 
 ******************
-Reference
+Quick Start
 ******************
 
+In this section, we're going to see how to build up a simple NStack module from scratch.
+We're going to build a small python module to greet people,
+and then hook it up to a source and a sink.
+
+Make sure you have :doc:`installed NStack </installation>` and let's get going.
+
```

Comment:
Ah, OK. I was thinking more that the Overview would be a prerequisite. Makes sense to make each piece standalone instead